distribution/std-lib/Standard/src/Image/Codecs/Internal.enso

```diff
@@ -0,0 +1,31 @@
+from Standard.Base import all
+import Standard.Image.Codecs
+
+polyglot java import org.opencv.core.MatOfInt
+polyglot java import org.opencv.imgcodecs.Imgcodecs
+
+## PRIVATE
```

Comment:
It would make much more sense to me to define these conversions as extension methods on the flags themselves. Something like `Codecs.Read_Alpha_Channel.to_integer = Imgcodecs.IMREAC_UNCHANGED`, and so on.